blockstore/apps/api/v1/views/bundles.py

```diff
@@ -12,6 +15,21 @@
 from ..serializers.bundles import BundleSerializer, BundleVersionSerializer, BundleVersionWithFileDataSerializer
 
 
+class BundleFilter(FilterSet):
+    """
+    Filter for BundleViewSet, which allows filtering by multiple at once UUIDs
+    """
+    uuid = AllValuesMultipleFilter(widget=CSVWidget)
+    search_text = CharFilter(method='search')
```

Comment:
Can you rename this to `text_search`? See https://openedx.atlassian.net/wiki/spaces/AC/pages/18350757/edX+REST+API+Conventions#edXRESTAPIConventions-4.URLParameters